docs/querying/querying.md

```diff
@@ -30,6 +30,8 @@ title: "Native queries"
  it runs a SQL query, refer to the [SQL documentation](sql-translation.md#query-types).
 :::
 
+For instructions on how the different approaches for set query context in your queries, see [Set the Query Context](./set-query-context.md).
```

Comment:
Recommend creating a learn more section to include this link. Also link to
* How to set query context
* Tutorial if there is an appropriate one